packages/components/src/public/QueryInfo.ts

```diff
@@ -171,6 +171,17 @@ export class QueryInfo {
         return this.appEditableTable && this.getPkCols().length > 0;
     }
 
+    getColumnFromName(name: string): QueryColumn {
+        if (name) {
+            const matchingCols = this.columns.filter(col => col.name === name).valueArray;
```

Comment:
the fieldKey check does toLowerCase() to make it case-insensitive. I agree that this likely should not do that so we are case sensitive on the name check. Just calling it out to verify. Thoughts?

Reply:
Why be case sensitive here? Feels like resolving the column is the intention of this function. It should be on callers to handle the casing within their usages (e.g. if they need to map it into another object).